nasa-gibs/worldview
# WV-2228: Dateline granule crossing

## Description

When creating granule layers, check to see if they meet certain conditions and shift them 360 degrees to the right when the dateline is crossed.

## How To Test

* Checkout `wv-2228-config` branch
* `npm run build`
* Checkout `wv-2228` branch
* `npm run watch`

Start with [this test URL](http://localhost:3000/?v=-249.08751893866332,-221.95387419213108,147.80705826510075,192.95313296297752&z=4&ics=true&ici=5&icd=6&l=VIIRS_NOAA20_CorrectedReflectance_TrueColor_Granule_v1_NRT,Coastlines_15m&lg=true&t=2019-09-23-T23%3A51%3A00Z)

1. Step forward in time until granules footprints cross the dateline
2. Confirm granules shift to the right
3. Step back until granules shift back again
4. Switch to a polar projection
5. Confirm granules and footprints look as expected
6. Switch back to Geographic projection
7. Confirm granules and footprints look as expected
